backend/README.md

```diff
@@ -1,6 +1,6 @@
 # Edna server
 
-Web server for Edna.
+Web server (backend) for Edna.
```

Comment:
Excuse me for being pedantic, but it's an API backend service, not a "web server".

Reply:
Fixed.

Reply:
By web server people usually mean anything serving http requests, a backend service matches this definition too